cluster-autoscaler/cloudprovider/digitalocean/digitalocean_node_group.go

```diff
@@ -233,9 +234,9 @@ func (n *NodeGroup) Autoprovisioned() bool {
 // toInstances converts a slice of *godo.KubernetesNode to
 // cloudprovider.Instance
 func toInstances(nodes []*godo.KubernetesNode) []cloudprovider.Instance {
-	instances := make([]cloudprovider.Instance, len(nodes))
-	for i, nd := range nodes {
-		instances[i] = toInstance(nd)
+	instances := []cloudprovider.Instance{}
```

Comment:
FWIW, you could use

```go
instances := make([]cloudprovider.Instance, 0, len(nodes))
```

to allocate the slice at the right size while also being able to use `append`.